resources/lang/en/pages.php

```diff
@@ -132,6 +132,8 @@
             'automatic_notice' => 'The QR-code is updated automatically, you do not need to press anything.',
             'specify_amount'   => 'Specify Amount',
             'send_from_wallet' => 'Send From Wallet',
+            'currency_amount'  => ':currency Amount',
+            'memo'             => 'Memo (optional)',
```

Comment:
```suggestion
            'memo_optional'             => 'Memo (optional)',
```
since the optional part is not optional in this string, makes it easier in case we need regular "memo" text at some point